src/main/java/com/xiaomi/infra/pegasus/client/PegasusClient.java

```diff
@@ -68,23 +64,26 @@ private PegasusTable getTable(String tableName, InternalTableOptions internalTab
     return table;
   }
 
+  private void initTableMap() {
+    this.tableMap = new ConcurrentHashMap<String, PegasusTable>();
+    this.tableMapLock = new Object();
+  }
+
   public PegasusClient(Properties properties) throws PException {
-    this(ClientOptions.create(properties));
+    super(properties);
+    initTableMap();
+    LOGGER.info(getConfigurationString());
   }
 
-  // configPath could be:
-  // - zk path: zk://host1:port1,host2:port2,host3:port3/path/to/config
-  // - local file path: file:///path/to/config
-  // - resource path: resource:///path/to/config
   public PegasusClient(String configPath) throws PException {
-    this(ClientOptions.create(configPath));
+    super(configPath);
+    initTableMap();
+    LOGGER.info(getConfigurationString());
```

Comment:
`LOGGER.info(getConfigurationString());` should put `abstract` and no need repeat it, include ` LOGGER.info(
        "Create PegasusAdminClient Instance By ClientOptions : {}", this.clientOptions.toString());` in `adminClient`